server.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask,render_template,request,redirect,url_for,flash,jsonify
+import requests
+
+from rss_generator import passer
+import json
+
+app=Flask(__name__)
+@app.route('/')
+def initial():
+    
+        return render_template('index.html',result=' ')
+    #else:
+        #return render_template('newmenu.html',restaurant_id=restaurant_id)  
+@app.route('/query', methods=['GET', 'POST'])
+def query_processor():
+        query=request.form['query']
```

Comment:
I see there is no connection made to the mongo database. Please implement that along with the changed schema. 
Where are you storing the query string?